<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-20T15:41:12 IST" name="Default suite" finished-at="2023-11-20T15:41:13 IST" duration-ms="510">
    <groups>
    </groups>
    <test started-at="2023-11-20T15:41:12 IST" name="Default test" finished-at="2023-11-20T15:41:13 IST" duration-ms="510">
      <class name="com.app.ListenerDemo.TestDemoListener">
        <test-method signature="method1()[pri:1, instance:com.app.ListenerDemo.TestDemoListener@40a4337a]" started-at="2023-11-20T15:41:13 IST" name="method1" finished-at="2023-11-20T15:41:13 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method2()[pri:2, instance:com.app.ListenerDemo.TestDemoListener@40a4337a]" started-at="2023-11-20T15:41:13 IST" name="method2" finished-at="2023-11-20T15:41:13 IST" duration-ms="32" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.app.ListenerDemo.TestDemoListener.method2(TestDemoListener.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method2 -->
      </class> <!-- com.app.ListenerDemo.TestDemoListener -->
      <class name="com.Ecommerce.qa.Base.TestBase">
        <test-method is-config="true" signature="InitalizeExtentReport()[pri:0, instance:com.Ecommerce.qa.Base.TestBase@4b2bac3f]" started-at="2023-11-20T15:41:11 IST" name="InitalizeExtentReport" finished-at="2023-11-20T15:41:12 IST" duration-ms="1220" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InitalizeExtentReport -->
        <test-method is-config="true" signature="getnameMethod(org.testng.ITestContext)[pri:0, instance:com.Ecommerce.qa.Base.TestBase@4b2bac3f]" started-at="2023-11-20T15:41:12 IST" name="getnameMethod" finished-at="2023-11-20T15:41:12 IST" duration-ms="272" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3a1d593e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getnameMethod -->
        <test-method is-config="true" signature="generateReports()[pri:0, instance:com.Ecommerce.qa.Base.TestBase@4b2bac3f]" started-at="2023-11-20T15:41:13 IST" name="generateReports" finished-at="2023-11-20T15:41:16 IST" duration-ms="3567" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReports -->
      </class> <!-- com.Ecommerce.qa.Base.TestBase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
